EDRSandblast/UserlandBypass/ProcessDumpDirectSyscalls.c

```diff
@@ -380,10 +380,15 @@ DWORD SandMiniDumpWriteDump(TCHAR* targetProcessName, WCHAR* dumpFilePath) {
 
     WCHAR FilePath[MAX_PATH] = { 0 };
     const WCHAR prefix[] = L"\\??\\";
-    memcpy_s(FilePath, sizeof(FilePath), prefix, sizeof(prefix));
-    UNICODE_STRING dumpFilePathAsUnicodeStr = { 0 };
-    wcscat_s(FilePath, _countof(FilePath), dumpFilePath);
+    const WCHAR driveLetterPrefix[] = L"C:";
```

Comment:
It seems a bit arbitrary, maybe something like `GetVolumePathNameW(L".", driveLetterPrefix, _countof(driveLetterPrefix))` instead ?

Reply:
hm yeah that's kind of arbitrary but I don't know about the GetVolumePathNameW. Say I'm running EDRSandblast from D:\ I wouldn't want it to try to write to d:\windows\temp\, I still want it to understand if I give a \windows\temp\my.dmp as a dump path. Could we just get rid of the \??\ prefix maybe ?

Reply:
Yes that would make sense. Another idea rather than adding a drive to the path, would you be ok with removing the `\??\` prefix ? That we we let windows resolve the path wether it contains the drive or not ?